templates/profile-base.html

```diff
@@ -4,6 +4,20 @@
     % set nofollow = ['nofollow'] if participant.profile_nofollow else []
     % set noindex = ['noindex'] if participant.profile_noindex or 'amount' in request.qs else []
     % if nofollow or noindex
-    <meta name="robots" content="{{ ','.join(noindex + nofollow) }}" />
+    <meta name="robots" content="{{ ','.join(noindex + nofollow) }}">
     % endif
+    <meta name="description" content="{{ _("%s's profile in Liberapay" % participant.username ) }}">
+    <meta name="og:title" content="{{ participant.username }}">
+    <meta name="og:type" content="profile">
+    <meta name="og:url" content="{{ website.canonical_scheme + '://' + request.hostname + '/' + participant.username }}">
+    <meta name="og:image" content="{{ participant.avatar_url }}">
+    <meta name="og:description" content="???">
+    <meta name="og:locale" content="{{ locale.language }}">
```

Comment:
`locale.language` is only constant in subdomains (e.g. <https://en.liberapay.com/>), on <https://liberapay.com/> it depends on the request header `Accept-Language`.